x-pack/legacy/plugins/siem/public/components/subtitle/index.tsx

```diff
@@ -35,9 +35,23 @@ interface SubtitleItemProps {
 
 const SubtitleItem = React.memo<SubtitleItemProps>(({ children }) => {
   if (typeof children === 'string') {
-    return <p className="siemSubtitle__item siemSubtitle__item--text">{children}</p>;
+    return (
+      <p
+        className="siemSubtitle__item siemSubtitle__item--text"
+        data-test-subj="header-panel-subtitle"
```

Comment:
maybe we can assign it to the variable and use it in both places?

Reply:
@patrykkopycinski can you please check that I did the changes in a correct way? Thx 😊